docs/dev/release_process.rst

```diff
@@ -105,6 +134,16 @@ Release with PyPI using the make command::
 
 Declare victory.
 
+Post-release TODO
+~~~~~~~~~~~~~~~~~
+
+* Release on PyPI
+* Update any redirects on learningequality.org for the latest release.
+* Announce release on dev list and newsletter if appropriate.
+* Close, if fixed, or change milestone of any issues on this release milestone.
+* Close this milestone.
+* For issues on this milestone that have been reported by the community, respond on the issues or other channels, notifying of the release that fixes this issues.
+
```

Comment:
This section seems a bit speculative since we haven't been doing much of it. I like the spirit of it, but maybe we should add a note that it's in-progress or a working draft?

Reply:
Yeah, this is intended to be prescriptive, not descriptive :)

Reply:
(and a lot of it is copied from the KA Lite release checklist)

Reply:
Added a note that this is aspirational :)